integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataETagTestCase.java

```diff
@@ -282,16 +276,16 @@ public void validateETagConcurrentHandlingTestCaseForDeleteMethod() throws Excep
     public void validateETagConcurrentHandlingTestCaseForUpdatePropertyWithPutMethod() throws Exception {
         // To insert values
         String endpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/FILES";
-        String content = "{\"FILENAME\": \"M.K.H.Gunasekara\" ,\"TYPE\" : \"dss\"}";
+        String content = "{\"FILENAME\": \"WSO2PROD\" ,\"TYPE\" : \"dss\"}";
         Map<String, String> headers = new HashMap<>();
         headers.put("Accept", "application/json");
         Object[] response = sendPOST(endpoint, content, headers);
         Assert.assertEquals(response[0], ODataTestUtils.CREATED);
-        String entityEndpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/FILES(\'M.K.H.Gunasekara\')";
-        response = sendGET(endpoint, headers);
+        String entityEndpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/FILES(\'WSO2PROD\')";
+        response = sendGET(entityEndpoint, headers);
```

Comment:
We have sent the request to **endpoint** earlier. Now we are sending it to **entityEndpoint**. Is this intentional?

Reply:
Yes. The idea of this testcase is to get etag of this particular field. But eventhough the url is set to filter this field, the previously defined url is used to invoke which gives all the fields. 

Earlier, the database scripts were ran per testcase. In that case, only the table is created and no rows are inserted. Therefore, as only one row is inserted before this line, only that field is getting retrieved. So the extracted etag matches exactly to this field. Therefore, this didn't fail. 

But now, we create the database after running all the scripts before running the test suite. Therefore, at this point there are multiple rows already available in the table. So the extracted etag does not exactly matches with this row.